ipatests/prci_definitions/gating.yaml

```diff
@@ -242,3 +242,15 @@ jobs:
         template: *ci-master-f28
         timeout: 3600
         topology: *master_1repl
+  
+  fedora-28/test_ntp_replacement_TestNTPreplacement:
```

Comment:
IMO we don't have to run this test cases on every commit. Gating tests must be fast. The NTP tests take way over an hour to run. Nightly tests should be sufficient, since NTP replacement isn't that critical.

Reply:
Sure, I will remove from gating.yaml